conf/dev.py

```diff
@@ -0,0 +1,6 @@
+DATABASE = 'neo4j-dev'
+DATABASE_HOST = 'localhost'
+DATABASE_USER = 'neo4j'
+DATABASE_PASS = 'neo5j'
+DATABASE_PORT = 7687
+DEBUG = True
```

Comment:
Both of these are pretty easy (neo4j is just download and run). What we really need is somewhere to host this nonsense...